.github/workflows/publish.yml

```diff
@@ -268,13 +268,16 @@ jobs:
 
   build-x86_64-apple-darwin-packages:
```

Comment:
We should probably update this job name if it starts building arm artifacts too.